packages/table/test/tableTests.tsx

```diff
@@ -1566,19 +1566,10 @@ describe("<Table>", () => {
             });
 
             describe("warnings", () => {
-                let consoleWarn: sinon.SinonSpy;
-
-                before(() => {
-                    consoleWarn = sinon.spy(console, "warn");
-                });
-
-                afterEach(() => {
-                    consoleWarn.reset();
-                });
-
-                after(() => {
-                    consoleWarn.restore();
-                });
+                let consoleWarn: sinon.SinonStub;
+                before(() => (consoleWarn = sinon.stub(console, "warn")));
+                afterEach(() => consoleWarn.resetHistory());
+                after(() => consoleWarn.restore());
```

Comment:
not a fan of these syntax changes, it's entirely unnecessary and makes the code too dense / hard to modify

Reply:
meh, prettier is good at fixing it for you

Reply:
what do you mean? what if I have to add multiple statements to each of these blocks? it's a lot more editing. there's no benefit to this code style